projects/1.18/assets/buddycards/buddycards/lang/zh_cn.json

```
@@ -1 +1,302 @@
-{}
\ No newline at end of file
+{
+	"block.buddycards.oak_card_display": "橡木卡片展示框",
+	"block.buddycards.birch_card_display": "白桦木卡片展示框",
+	"block.buddycards.spruce_card_display": "云杉木卡片展示框",
+	"block.buddycards.jungle_card_display": "丛林木卡片展示框",
+	"block.buddycards.dark_oak_card_display": "深色橡木卡片展示框",
+	"block.buddycards.acacia_card_display": "金合欢木卡片展示框",
+	"block.buddycards.warped_card_display": "诡异木卡片展示框",
+	"block.buddycards.crimson_card_display": "绯红木卡片展示框",
+	"block.buddycards.buddycard_booster_box_mystery": "神秘 卡牌包组",
+	"block.buddycards.buddycard_booster_box_base": "基础 卡牌包组",
+	"block.buddycards.buddycard_booster_box_nether": "下界 卡牌包组",
+	"block.buddycards.buddycard_booster_box_end": "末地 卡牌包组",
+	"block.buddycards.buddysteel_block": "巴迪钢块",
+	"item.buddycards.buddycard_pack_base": "基础 卡牌包",
+	"item.buddycards.buddycard_pack_nether": "下界 卡牌包",
+	"item.buddycards.buddycard_pack_end": "末地 卡牌包",
+	"item.buddycards.buddycard_pack_mystery": "神秘 卡牌包",
+	"item.buddycards.buddycard_binder_base": "基础 卡牌夹",
+	"item.buddycards.buddycard_binder_nether": "下界 卡牌夹",
+	"item.buddycards.buddycard_binder_end": "末地 卡牌夹",
+	"item.buddycards.ender_buddycard_binder": "末影 卡牌夹",
+	"item.buddycards.buddysteel_blend": "巴迪钢粉",
+	"item.buddycards.buddysteel_ingot": "巴迪钢锭",
+	"item.buddycards.buddysteel_nugget": "巴迪钢粒",
+	"item.buddycards.buddysteel_power_meter": "巴迪钢卡牌检测仪",
+	"item.buddycards.buddysteel_power_meter.desc": "§7检查您的卡组收藏进度, 或者将卡片放在副手栏位中以检查特定卡组的收藏进度",
+	"item.buddycards.buddysteel_power_meter.total_cards_collected": "收藏的巴迪卡总数：",
+	"item.buddycards.buddysteel_power_meter.cards_collected": "已收藏的巴迪卡：",
+	"item.buddycards.blank_buddysteel_medal": "空白巴迪钢勋章",
+	"item.buddycards.blank_buddysteel_medal.desc": "§7将一张来自已完成收藏的卡组中的一张卡牌放在副手上, 然后右键单击, 来对勋章充能",
```

```suggestion
	"item.buddycards.blank_buddysteel_medal.desc": "§7将一张来自已完成收藏的卡组中的一张卡牌放在副手上，然后右击来对勋章充能",
```

标点符号应使用规范的汉语全角标点符号，具体要求请阅读我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **03.标点符号** 这一节。
诸如这类的问题请在您翻译的所有文件中全部查明并修改。